com.unity.cinemachine/Editor/Editors/CinemachineConfiner2DEditor.cs

```diff
@@ -33,48 +34,69 @@ public override VisualElement CreateInspectorGUI()
 
             var volumeProp = serializedObject.FindProperty(() => Target.BoundingShape2D);
             ux.Add(new PropertyField(volumeProp));
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.Damping)));
-            var oversizedCameraHelp = ux.AddChild(new HelpBox(
-                "The camera window is too big for the confiner. Enable the Oversize Window option.",
-                HelpBoxMessageType.Info));
-            
-            ux.Add(new PropertyField(serializedObject.FindProperty(() => Target.OversizeWindow)));
-            
             TrackVolume(volumeProp);
             ux.TrackPropertyValue(volumeProp, TrackVolume);
             void TrackVolume(SerializedProperty p)
             {
                 var c = p.objectReferenceValue;
-                boundsHelp.SetVisible(!(c is PolygonCollider2D || c is BoxCollider2D || c is CompositeCollider2D));
+                boundsHelp.SetVisible(!(c is null || c is PolygonCollider2D || c is BoxCollider2D || c is CompositeCollider2D));
```

Comment:
Why this change?  Doesn't it make sense to have the help message if the shape is null also?

Reply:
I felt no, but you are right, it makes sense to show it when null.